### DISCUSSION TITLE
2023-10-04 Weekly Meeting 127 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#420] [Weekly Meetings]
Weekly Meeting 127 is happening on Wednesday, October 4, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in the 🔈 omi-weekly-meeting chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda:

1.   "420".
2. Oopsie! where is last week's meeting? (Link below but not visible in the [discussions page](https://github.com/omigroup/omigroup/discussions))? Is this a bug or did a lot happen in the backlog refinement?
3. Feedback on metatraversal
4. Feedback on OMI demo day "interoperable physics in the open metaverse"
5. Monthly OMI demo days: schedule and take ownership. [Hackmd ](https://hackmd.io/@indiebio/OMI-monthly-demo-days)for scheduling and [discussion thread on Discord](https://discord.com/channels/770382203782692945/1149030210503716944).
6. Weekly game nights? 
  e.g. foosball, darts, card games, and beanbags, and a little movie theatre where we show movies. Then we meet there, like a clubhouse, once a week. No agenda, just chillin' together. Maybe in VR chat?
7. Elections.
There will be elections for OMI chairs. MrMetaverse is up for re-election, indiebio has nominated herself for election.
Nominations will be open until <date>
Election will be on <date>
More info on roles of Chairs https://hackmd.io/@mrmetaverse/omi-tasks-audit; 
also: https://github.com/omigroup/omigroup/discussions/157
and https://github.com/omigroup/omigroup/discussions/165
8. EU research funding cycle open. Do a demo day to see what is out there, match up with people? [Thread on Discord](https://discord.com/channels/770382203782692945/1158355479349702686)
9. Question in Discord about "[different implementations of metaverse engines](https://discord.com/channels/770382203782692945/895065600353792081/1158043241158029402)" - this could be a great demo day topic as well? Or some sort of a thing. Like a youtube debate.
10. OMF collaboration ... should we try work this? As a separate but closely aligned partner, perhaps the commercial side to OMI's grassroots community? 
10.1. Where should OMF focus on (e.g. what working groups)
10.2 Where should the closest collaboration between  OMF and OMI be?
11. Media group needs more integration!

Working group check-ins

- Revisit sending an invitation about growing the glTF spec group, to students

Last week: https://github.com/omigroup/omigroup/discussions/419

**Comment by indiebio** [comment]
3. metatraversal had a good turnout, was a rapid turnover, visited a lot of places. Opportunities to collaborate.
Keyframe was part of the production. Was good, could have been more seamless with integration with Zoom. Also had streamyard going for presentation flow which also showed bottom-third. Integration could have been better, being improved going forwards. VRLand and Upland are creating portal implementations (?).
Keyframe is processing video this week, needs to be shorter than the whole 1hour and will share next week.

**Reply by indiebio** [reply]
keyframe to share notes on good spaces of virtual spaces.
20 October launch of museum, can do demo day on that date too, work together. Crowdfunder.

More info on our digital museum project https://www.virtualworlds.museum/  crowdfunder launches 10/20/23
Museum open 2024

**Comment by indiebio** [comment]
4. OMI demo day was good, someone was demo'ing camera control system. Post-production live in event sort of thing, programmatically move cameras and adjust focal length etc.
jin pulled it off, M3 put in a great effort

**Reply by indiebio** [reply]
https://cdn.discordapp.com/attachments/839486903178362920/1159162218907172874/Screen_Recording_20230930_120035_Brave.mp4?ex=651ee171&is=651d8ff1&hm=10dfb80d202ea20ba9731b430aac9cbdd5d1d5c18718a22aebe8a366980fbaf8&

**Reply by indiebio** [reply]
jin made notes in the meeting channel, there were two people recording, and it was transcribed with ai. People who attending has a photowall to commemorate the event.
https://hackmd.io/@XR/OMI_demoday-antpb
![image](https://github.com/omigroup/omigroup/assets/23474368/52bff042-4447-4781-b2de-9f532419b867)

**Comment by indiebio** [comment]
5. Demo day scheduling, "Wouldn't it be cool if" demo day ... and then we can talk, who is interested, who is doing this, what are the challenges?

**Comment by indiebio** [comment]
6. Movie nights not as good, but background music can work. Being together in a space is good.
Can have crash-cars for those who want to move around.
Metaverse makerspace?

**Comment by indiebio** [comment]
7. Give enough notice on nominations and elections in the coming meetings too. Will update the dates ASAP.

**Comment by indiebio** [comment]
9. All metaverse **engines** have in common to take your assets with you.
So VRchat could be classified as one.
A game engine could be *used* to build a metaverse platform but it is not a engine/platform in itself.
> Need a channel or a FAQ or something to deal with the terminology as it emerges.

**Comment by indiebio** [comment]
10. Could align some of our sessions with OMF. Could allow a mixing zone for conversations to take place...

**Reply by jimmy6DOF** [reply]
![20231010_1129_POWERPNT_9n8K6FfQEN](https://github.com/omigroup/omigroup/assets/89285877/0d0ceb66-70de-40a1-8e9e-e1c4d451111d)

This is update from the workgroup (Plenary # 8 - Sep 13) about MSF's reference library that was  **not**  launched in the Metaverse.  [It's a repository for standards register, glossary, and use cases.  Currently at 64 entries.](https://register.metaverse-standards.org/?q=&candidates=true)

**Comment by indiebio** [comment]
Working groups:
glTF last week worked on audio stuff.
thanks to antpb for spearheading that.

**Reply by indiebio** [reply]
Currently we are listing example implementers for our specs.
Would it be good to list all the metaverse platforms and what specs they use?
Yes!
How would that look? Would it be a website, a github repo, a virtual space?
At the least do it in markdown. Can then visualise it on top of that.

related: (Keyframe) Some WebXR platforms I’ve been tracking https://github.com/KEYFRAME-SEHU/mapping

**Comment by indiebio** [comment]
Proposal submissions open for Interop2024: https://web.dev/interop-2024-proposals/

**Comment by indiebio** [comment]
11. Media group needs calendar added, permissions topost / update website

**Comment by indiebio** [comment]
MSF is creating a library of reference material. sortof adjacent to the museum concept. jimmy to give link for more info.
Could be an opportunity to collaborate in the display or access of these.
